daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -284,8 +289,39 @@ To publish a message to a topic, invoke the following endpoint on a Dapr instanc
 {{< tabs "Dapr CLI" "HTTP API (Bash)" "HTTP API (PowerShell)">}}
 
 {{% codetab %}}
+To use the Dapr CLI, you need to have an application running that has the permissions to publish topics. Create a new file called `pubapp.js` with the following code.
+
+```javascript
+const express = require('express');
+const path = require('path');
+const request = require('request');
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(bodyParser.json());
+
+const daprPort = process.env.DAPR_HTTP_PORT || 3500;
+const daprUrl = `http://localhost:${daprPort}/v1.0`;
+const port = 8080;
+const pubsubName = 'pubsub';
+
+app.post('/publish', (req, res) => {
+  console.log("Publishing: ", req.body);
+  const publishUrl = `${daprUrl}/publish/${pubsubName}/deathStarStatus`;
+  request( { uri: publishUrl, method: 'POST', json: req.body } );
+  res.sendStatus(200);
+});
+
+app.listen(process.env.PORT || port, () => console.log(`Listening on port ${port}!`));
+```
+Start the pubapp running Dapr by
+
 ```bash
-dapr publish --pubsub pubsub --topic deathStarStatus --data '{"status": "completed"}'
+dapr --app-id pubapp --port 3500 node run pubapp.js
```

Comment:
I always find it confusing when the CLI sub-command (in this case "run") does not come as the first argument. Even though this may work I think it is clearer in the docs to use "dapr run ..." consistently.
(in this file there are actually additional places where this happens)

Reply:
Agreed. I changed all of the occurrences